index.php

```diff
@@ -59,5 +59,10 @@
         }
         ?>
     </ol>
+
+    <form>
+        <input type="text" name="pridej">
```

Comment:
To je matoucí jak je to "pridej" a pak je tu button "Přidat".

Nazval bych "novy", "dalsi" nebo tak něco. Jo a napsat před to o co jde, ne?

```html
Jméno: <input type="text" name="jmeno"><br>
<input type="submit" value="Přidat">
```